index.d.ts

```diff
@@ -0,0 +1,26 @@
+type IPv4Address = string;
+type IPv4CIDR = string;
+type IPv6Address = string;
+type IPv6CIDR = string;
+
+type Network = IPv4Address | IPv4CIDR | IPv6Address | IPv6CIDR;
+
+type Networks = Network | Network[];
+
+interface CIDRTools {
+  merge(networks: Networks): Network[];
+
+  exclude(baseNetworks: Networks, excludeNetworks: Networks): Network[];
+
+  expand(networks: Networks): Network[];
+
+  overlap(networksA: Networks, networksB: Networks): boolean;
+
+  normalize(Network): Network;
```

Comment:
I think you can do without the empty lines between the methods.

Reply:
Done (rebased).

Great tool by the way. I needed to turn an acceptlist into a rejectlist:

```js
const rejectList = cidrTools.exclude('0.0.0.0/0', acceptList);
```

(Works the same the other way too.)

Reply:
Care to answer my other question in the issue comments? I have no experience with TypeScript, but I guess definitions must be published along with the module so tools can pick them up from `node_modules`?